examples/integration/BUILD

```diff
@@ -6,6 +6,10 @@ load(":xcodeproj_targets.bzl", "CONFIG", "XCODEPROJ_TARGETS")
 
 xcodeproj(
     name = "xcodeproj",
+    associated_extra_files = {
+        "//iOSApp/Source:iOSApp": ["//iOSApp:ownership.yaml"],
+        "//Lib": ["//Lib:README.md"],
+    },
```

Comment:
My comment was lost, but this should be set in `//:xcodeproj_targets.bzl` and reused here and in the fixture target.